### ISSUE TITLE
Duplicated sections

### ISSUE THREAD

**Issue by Tinche** [nedbat/watchgha#19]
Howdy,

it seems my workflows are displayed twice?

```
msgspec support tin/msgspec [pull_request]   a973cae81ae1  @01:19am
   ↻ in_progress  CI                 view 7496142219
      Python 3.8                     ↻ ••••••• Run Tox
      Python 3.9                     ↻ ••••••• Run Tox
      Build & verify package         ✓ success
      Python 3.10                    ↻ ••••••• Run Tox
      Python 3.11                    ↻ ••••••• Run Tox
      Python 3.12                    ✓ success
      Python pypy-3.8                ✓ success
   ↻ in_progress  CI                 view 7496142219
      Python 3.8                     ↻ ••••••• Run Tox
      Python 3.9                     ↻ ••••••• Run Tox
      Build & verify package         ✓ success
      Python 3.10                    ↻ ••••••• Run Tox
      Python 3.11                    ↻ ••••••• Run Tox
      Python 3.12                    ✓ success
      Python pypy-3.8                ✓ success
```
The run IDs seem to be the same.

**Comment by nedbat** [comment]
That's very odd.  Is that repo public? Can I see the actions?

**Comment by Tinche** [comment]
Sure, it's https://github.com/python-attrs/cattrs.

**Comment by nedbat** [comment]
The action run itself doesn't look different.  Does it always happen?

**Comment by Tinche** [comment]
I haven't been using the tool a lot but for now I only see the issue on this repository. I tried it on some work repos and those seem fine. But on this repo, both PRs I've been working on display it.

**Comment by nedbat** [comment]
Hmm, when I look at that branch in your repo, I get only one:
```
% watch_gha_runs https://github.com/python-attrs/cattrs tin/msgspec
msgspec support tin/msgspec [pull_request]   a973cae81ae1  @07:19pm
   ✓ success      CI                 view 7496142219
```
What output do you get for `git remote -v` ?

**Comment by Tinche** [comment]
If I look at the branch, I only get one:

```
❯ watch_gha_runs https://github.com/python-attrs/cattrs tin/msgspec
msgspec support tin/msgspec [pull_request]   a973cae81ae1  @01:19am
   ✓ success      CI                 view 7496142219
```

But if I just do `watch_gha_runs`:
```
❯ watch_gha_runs
Guess we doin V now (a new validation framework) tin/so-much-v [pull_request]   9b7cbc14f25e  @01:37am
   ✓ success      CI                 view 7496305052
   ✓ success      CI                 view 7496305052
```

> What output do you get for git remote -v ?

```
❯ git remote -v
origin	git@github.com:python-attrs/cattrs.git (fetch)
origin	git@github.com:python-attrs/cattrs.git (push)
python-attrs	git@github.com:python-attrs/cattrs.git (fetch)
python-attrs	git@github.com:python-attrs/cattrs.git (push)
```

Interesting, I have duplicated remotes for some reason. Probably historical. If I remove one, that fixes the issue. Thanks!

**Comment by nedbat** [comment]
Fixed in 40af699f6caee94029500508531c93f61b974225.

**Comment by nedbat** [comment]
This is now released as part of [watchgha 2.2.1](https://pypi.org/project/watchgha/2.2.1).
